src/main/java/it/pagopa/pm/gateway/client/bpay/BancomatPayClient.java

```diff
@@ -29,6 +29,7 @@ public class BancomatPayClient {
 
     private final ObjectFactory objectFactory = new ObjectFactory();
 
+    @SuppressWarnings("unchecked")
```

Comment:
Please include a comment detailing why this annotation is necessary

Reply:
it's not strictly necessary, it's just to suppress some useless warnings regarding XML casting.

Reply:
(Either add this or remove the `@SuppressWarning`s, and keep in mind to do so also to the other instances)

Reply:
annotations removed.